polars_xdt/namespace.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from typing import Any, Callable
+
+import polars as pl
+
+from polars_xdt import functions
+
+
+@pl.api.register_expr_namespace("xdt")
+class ExprXDTNamespace:
+    """eXtra stuff for DateTimes."""
+
+    def __init__(self, expr: pl.Expr) -> None:
+        self._expr = expr
+
+    def __getattr__(self, function_name: str) -> Callable[[Any], pl.Expr]:
+        def func(*args: Any, **kwargs: Any) -> pl.Expr:
+            return getattr(functions, function_name)(
+                self._expr, *args, **kwargs
+            )
+
+        return func
```

Comment:
adding this for backwards-compatibility, so that
```python
from datetime import date

import polars as pl
import polars_xdt  # noqa: F401


df = pl.DataFrame(
    {"date": [date(2023, 4, 3), date(2023, 9, 1), date(2024, 1, 4)]}
)

result = df.with_columns(
    date_shifted=pl.col("date").xdt.offset_by(
      '5bd',
      weekend=('Sat', 'Sun'),
    )
)
print(result)
```

Not that there's many (any?) real users yet, but still, being backwards compatible is a good practice